ci-operator/step-registry/openshift-qe/hypershift-infra/openshift-qe-hypershift-infra-commands.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
```

Comment:
@yasun1 Do you see anyone else benefiting from having infra nodes for Hypershift cluster and moving pods for certain workloads to infra nodes? 

If yes, we can move it to higher level folder https://github.com/openshift/release/tree/5480ad055929b588e46f7db1df851c356278399a/ci-operator/step-registry/rosa/aws/sts/hypershift 
Let us know your thoughts!

Reply:
There is no Infra node for the rosa hypershift cluster. It is by design.